src/main/java/seedu/address/model/project/Project.java

```diff
@@ -146,6 +146,14 @@ public void removeTask(Task task) {
         tasks.remove(task);
     }
 
+    /**
+     * Sorts tasks in tasks list according to completion and deadline.
+     * Incomplete tasks will appear before complete tasks and tasks will be arranged in ascending deadline order.
+     */
+    public void sortTasksByDeadlineAndCompletion() {
```

Comment:
Given that you've implemented this new method, do you think we could maintain the tasks of a project in a sorted state upon deletion/addition of tasks? Though that would sort of eliminate the need for the `sortT` command...

Reply:
I was thinking of doing that actually, but I realised it could be best to not auto-sort since the users may be using the indexes for subsequent commands (and auto-sorting them could potentially mix those up), so giving them a choice of when to sort could be better